contracts/examples/AyiiProduct.sol

```diff
@@ -167,18 +171,51 @@ contract AyiiProduct is
             applicationData);
 
         _applications.push(processId);
+        EnumerableSet.add(_policies[riskId], processId);
+
+        emit LogAyiiPolicyApplicationCompleted(
```

Comment:
consider to rename to LogAyiiPolicyApplicationCreated to be consistent with event LogAyiiPolicyCreated